src/SAML2/XML/IdentifierTrait.php

```diff
@@ -42,7 +42,7 @@ public function getIdentifier(): ?IdentifierInterface
     /**
      * Set the value of the identifier-property
      *
-     * @param \SAML2\XML\saml\IdentifierInterface|null
+     * @param \SAML2\XML\saml\IdentifierInterface|null $identifier
```

Comment:
This has been fixed in master already.. Perhaps a rebase against master would be a good idea?

Reply:
It is!